README.md

```diff
@@ -441,9 +441,12 @@ This repository contains a curated list of awesome open source libraries that wi
 ## Neural Search
 * [Annoy](https://github.com/spotify/annoy) ![](https://img.shields.io/github/stars/spotify/annoy.svg?style=social) - Annoy (Approximate Nearest Neighbors Oh Yeah) is a C++ library with Python bindings to search for points in space that are close to a given query point.
 * [CLIP-as-service](https://github.com/jina-ai/clip-as-service) ![](https://img.shields.io/github/stars/jina-ai/clip-as-service.svg?style=social) - CLIP-as-service is a low-latency high-scalability service for embedding images and text. It can be easily integrated as a microservice into neural search solutions.
+* [DocArray](https://github.com/docarray/docarray) ![](https://img.shields.io/github/stars/docarray/docarray.svg?style=social) - DocArray is a library for nested, unstructured, multimodal data in transit, including text, image, audio, video, 3D mesh, etc. It allows deep-learning engineers to efficiently process, embed, search, recommend, store, and transfer multimodal data with a Pythonic API.
 * [Faiss](https://github.com/facebookresearch/faiss) ![](https://img.shields.io/github/stars/facebookresearch/faiss.svg?style=social) - Faiss is a library for efficient similarity search and clustering of dense vectors.
-* [NMSLIB](https://github.com/nmslib/nmslib) ![](https://img.shields.io/github/stars/nmslib/nmslib.svg?style=social) - Non-Metric Space Library (NMSLIB): An efficient similarity search library and a toolkit for evaluation of k-NN methods for generic non-metric spaces.
+* [Finetuner](https://github.com/facebookresearch/faiss) ![](https://img.shields.io/github/stars/facebookresearch/faiss.svg?style=social) - Jina AI's Finetuner makes fine-tuning easier and faster by streamlining the workflow and handling all the complexity and infrastructure in the cloud.
 * [NGT](https://github.com/yahoojapan/NGT) ![](https://img.shields.io/github/stars/yahoojapan/NGT.svg?style=social) - NGT provides commands and a library for performing high-speed approximate nearest neighbor searches against a large volume of data in high dimensional vector data space.
+* [NMSLIB](https://github.com/nmslib/nmslib) ![](https://img.shields.io/github/stars/nmslib/nmslib.svg?style=social) - Non-Metric Space Library (NMSLIB): An efficient similarity search library and a toolkit for evaluation of k-NN methods for generic non-metric spaces.
+* [NOW](https://github.com/jina-ai/now) ![](https://img.shields.io/github/stars/jina-ai/now.svg?style=social) - NOW is a no-code tool for creating a neural search solution in minutes.
```

Comment:
Seems more of an app/showcase creation than productionisation, there's quite a few jina tools already so can remove or group these

Reply:
NMSLIB and NOW have catch the attention of the industry recently, as seen by the rising stars up to 3k. @axsaucedo

Reply:
NMSLIB  is ok - my ocmment was on NOW which is another jina tool irt grouping these

Reply:
> NMSLIB is ok - my ocmment was on NOW which is another jina tool irt grouping these

According to the current standard of tool inclusion, then NOW seems to be ok to be included (>500 stars, is a tool that accelerates the development of ML models). If we really do not want to include a tool that facilitates the deployment of ML algorithms, then we need to change our inclusion bar accordingly. @axsaucedo

Reply:
This requsting grouping not removing + this is no-code tool as opposed to neural search

Reply:
> This requsting grouping not removing + this is no-code tool as opposed to neural search

No-code tools are too broad and there fails to exist a new section out of it too. Also, NOW is designed exclusively for neural search solution, maybe we should revisit https://github.com/EthicalML/awesome-production-machine-learning/issues/387 and see what is the standard for single categorization if a tool can be used for multi-purposes @axsaucedo What do you think?

Reply:
But we do not have no-code sections in our list, and NOW is indeed a tool designed exclusively for neural search. I do think this tool should fall into this section at this moment. @axsaucedo